addons/scene_palette/components/scene_thumbnail/scene_drop.gd

```diff
@@ -41,13 +53,9 @@ func set_scene(path:String):
 		'png':
 			texture_rect.texture = load(_scene_path)
 		'tscn':
-			if instantiate_scene_preview:
-					var node:Node = load(_scene_path).instantiate()
-					if _scene_is_safe(node):
-						picture_point.add_child(node)
-						return
-			# if scene is not safe to instantiate, just keep a preview
-			_make_preview()
+			_make_preview_for_scene()
+		'scn':
+			_make_preview_for_scene()
 		'obj':
 			_make_preview()
```

Comment:
I think either the name of `_make_preview`  or `_make_preview_for_scene` should be changed so they are less ambiguous.

Perhaps `_make_preview` could just be renamed `_make_default_editor_preview`?

Reply:
Sorry I can't leave auto suggestions since _make_preview wasn't in the PR

Reply:
Alternatively, you could just combine the scn and tscn cases on the same line as `'tscn', 'scn':` and not have to factor out a function at all